src/Altinn.Broker.Core/Domain/Enums/ActorFileStatus.cs

```diff
@@ -1,8 +1,9 @@
 namespace Altinn.Broker.Core.Domain.Enums;
 
-public enum ActorFileStatus {
+public enum ActorFileStatus
+{
     None = 0,
     Initialized = 1,
```

Comment:
Please update these in separate PR as agreed; https://github.com/Altinn/altinn-broker/issues/164